src/components/TileGroup/TileGroup.js

```diff
@@ -106,11 +111,22 @@ export default class TileGroup extends React.Component {
     }
   };
 
+  renderLegend = legend => {
+    if (legend) {
+      return <legend>{legend}</legend>;
```

Comment:
Does this node need to be styled at all?

Reply:
This will inherit the style from the parent element.